drivers/led/led_shell.c

```diff
@@ -85,6 +85,27 @@ static int cmd_on(const struct shell *sh, size_t argc, char **argv)
 	return err;
 }
 
+static int cmd_blink(const struct shell *sh, size_t argc, char **argv)
+{
+	const struct device *dev;
+	uint32_t led;
+	int err;
+
+	err = parse_common_args(sh, argv, &dev, &led);
+	if (err < 0) {
+		return err;
+	}
+
+	shell_print(sh, "%s: blinking LED %d", dev->name, led);
+
+	err = led_blink(dev, led);
```

Comment:
Did you test this ? `delay_on` and `delay_off` parameters are missing.

Reply:
Wow awkward. My bad. Did this in a hurry.